SPECS/usrsctp/usrsctp.signature.json

```diff
@@ -0,0 +1,5 @@
+{
+ "Signatures": {
+  "0.9.5.0.tar.gz": "260107caf318650a57a8caa593550e39bca6943e93f970c80d6c17e59d62cd92"
```

Comment:
How this signature generated ?

Reply:
> How this signature generated ?

By using `sha256sum <tarball>` we can generate the signature.